src/index.html

```diff
@@ -4,9 +4,286 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home" class="header__logo">
+        <img
+          src="./images/logo.png"
+          alt="logo"
+          class="logo"
+        >
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__list-item">
+            <a class="nav__link is-active" href="#apple">Apple</a>
+          </li>
+
+          <li class="nav__list-item">
+            <a class="nav__link" href="#samsung">Samsung</a>
+          </li>
+
+          <li class="nav__list-item">
+            <a class="nav__link" href="#smartphones">Smartphones</a>
+          </li>
+
+          <li class="nav__list-item">
+            <a
+              class="nav__link"
+              data-qa="nav-hover"
+              href="#laptops"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="nav__list-item">
+            <a class="nav__link" href="#gadgets">Gadgets</a>
+          </li>
+
+          <li class="nav__list-item">
+            <a class="nav__link" href="#tablets">Tablets</a>
+          </li>
+
+          <li class="nav__list-item">
+            <a class="nav__link" href="#photo">Photo</a>
+          </li>
+
+          <li class="nav__list-item">
+            <a class="nav__link" href="#video">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main">
+      <ul class="main__grid-container">
+        <li class="main__grid-item">
+          <div data-qa="card" class="card">
+            <img
+                src="./images/imac.jpeg"
+              alt="APPLE A1419 iMac 27&#34 Retina 5K Monoblock (MNED2UA/A)"
+              class="card__image"
+            >
+            <h3 class="card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h3>
+            <p class="card__subtitle">Product code: 195434</p>
```

Comment:
```suggestion
            <h3 class="card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h3>
            
            <p class="card__subtitle">Product code: 195434</p>
```

Reply:
<img width="1071" alt="image" src="https://github.com/mate-academy/layout_catalog/assets/86789161/256b4f3e-9ac6-4f97-beb9-81dbaac61793">
Fix in all places